### ISSUE TITLE
too big message problem/RTMP in chunk stream too big

### ISSUE THREAD

**Issue by Lotroj** [arut/nginx-rtmp-module#576]
Hi, 

i have problem with this error msg.

Nginx version 1.4.4-1 + actual rtmp modul compiled under debian, runing on openwrt based router Turris, https://www.turris.cz/en/

Changing max_message doesnt have effect.
Error occurs on "both ways" when i try streaming to server (using OBS and Xsplit) or when i try recieve stream from server (using this https://dl.dropboxusercontent.com/u/2918563/flvplayback.swf)

At max_message 50m:
Error log, when trying stream to server:
http://pastebin.com/qcjpxb0A

Error log, when try recieve stream:
http://pastebin.com/pu8SekgY

At max_message 500m:
streaming log:
http://pastebin.com/PJ3mJkE4

recieving log:
same as at max_message 50n

Config:
http://pastebin.com/SKAXAsWC (yes i know i paste my twitch key, i reset it already)

Thanks for any help

**Comment by ametatsu** [comment]
Sorry for reviving this thread, but I'm having the exact same issue here. Any solutions to this? Stream keeps restarting itself due to this error.

**Comment by Lotroj** [comment]
Hi,

no i never solved this and never received any reaction from author.

**Comment by rayj00** [comment]
I am having a "too big RTMP chunk size:134217728, client: 10.59.42.19:1935/live, server: ngx-relay" issue.
I cannot get a response from the author either.

Has anyone seen this error and how to fix?

Thanks,
Ray

**Comment by jakubgs** [comment]
Same here, I'm getting:
```
too big RTMP chunk size:134217728, client: livepeer:1935/, server: ngx-relay
```
And no matter how I select options it just keeps failing.

**Comment by wellttllew** [comment]
FYI:  

- [The author's reply: Change max_message config](https://github.com/arut/nginx-rtmp-module/issues/324)  
- [About  max_message directive](https://github.com/arut/nginx-rtmp-module/wiki/Directives#max_message)

**Comment by jakubgs** [comment]
Yeah, I went as low as setting `max_message` to `1` byte and it still would tell me:
```
[alert] 6#6: *11 too big RTMP chunk size:134217728, client: livepeer:1935/, server: ngx-relay
[info] 6#6: *11 too big message: 1, client: livepeer:1935/, server: ngx-relay
[info] 6#6: *11 disconnect, client: livepeer:1935/, server: ngx-relay
[info] 6#6: *11 deleteStream, client: livepeer:1935/, server: ngx-relay
```

**Comment by wellttllew** [comment]
@jakubgs

**Comment by jakubgs** [comment]
@wellttllew

**Comment by jbubik** [comment]
To anyone reading this in the future:

The original code is heavily little-endian "minded". ARUT's big-endian branch fixes some but not all issues. I tried to fix it properly. My PR is in repository @ut0mt8/nginx-rtmp-module - that is where OpenWrt currently takes sources from. Should get its way to OpenWrt packages eventually.
